azure-pipelines.yml

```diff
@@ -66,14 +101,13 @@ jobs:
       DOCKERHUB_PASSWORD: $(DockerPassword)
       GCP_SERVICE_ACCOUNT_KEY: $(GcpServiceAccountKey)
 
-- job: generate_toolchains_linux
-  dependsOn: "build_container_linux"
+- job: generate_toolchains_windows
```

Comment:
can this two generate_toolchains be templated/parameterized?

Reply:
made an attempt in the latest commit